README.markdown

```diff
@@ -15,8 +15,13 @@ PythonTurtle was created by Ram Rachum as a side-project in 2009. I also provide
 
 Installing on Linux:
 
+    # Ubuntu:
     sudo apt-get install python-wxgtk3.0 git -y
     # NOTE: install `python-wxgtk2.8` on olders systems
+
+    # Fedora:
+    dnf install wxPython
```

Comment:
`git` is missing in the package list, I guess. And what about `sudo`, doesn't Fedora favor this, too?